alephium/deploy.ts

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+import yargs from 'yargs/yargs';
+import { hideBin } from "yargs/helpers"
+
+import configuration from "./configuration"
+import { deploy, NetworkType } from "./lib/deployment"
+
+const argv = yargs(hideBin(process.argv))
```

Comment:
Nice :+1: 

Not related to this PR. If we wanna share this code across different projects, would it make sense to have a separate CLI for this? just like hardhat, brownie or [dapp.tools](https://github.com/dapphub/dapptools/blob/master/src/dapp/README.md#basic-usage-a-tutorial)? i guess other common tasks are: `compile`, `test`, `node`,  `deploy`, etc

Reply:
Yes, I think we also need to support these commands like truffle/hardhat. I'll add a separate CLI later.

Reply:
Nice :+1: 

Where do you plan to put the CLI? Should it be part of the web3 package or a separate repo? I guess the benefit of putting it in the same repo as web3 is that devs can be sure that the CLI version is the same as what the user specified in `package.json`, perhaps by using npx. WDYT?

Reply:
Yes, it should be part of the alephium-web3, users can use `npx alephium deploy/test/etc` like hardhat/truffle, I'll test with bridge first, and when it's ready I'll add it to the SDK.

Reply:
Awesome :+1: